snapshotTestContent/SharedMatrixDeletes/current_snapshots/snapshot_473_0.json

```diff
@@ -3633,7 +3633,7 @@
":[\"/490e917c-9805-40b5-93d9-3b52c444f8c8\"],\"unreferencedTimestampMs\":1646688495162},\"/490e917c-9805-40b5-93d9-3b52c444f8c8\":{\"outboundRoutes\":[\"/490e917c-9805-40b5-93d9-3b52c444f8c8/root\",\"/490e917c-9805-40b5-93d9-3b52c444f8c8/cf847818-809d-41f4-9dc8-e9389d87b82d\"],\"unreferencedTimestampMs\":1646688495162},\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b/root\":{\"outboundRoutes\":[\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b/2f989fd2-bb56-4d76-809e-b672000fa9f7\",\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b\"],\"unreferencedTimestampMs\":1646688495162},\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b/2f989fd2-bb56-4d76-809e-b672000fa9f7\":{\"outboundRoutes\":[\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b\"],\"unreferencedTimestampMs\":1646688495162},\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b\":{\"outboundRoutes\":[\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b/root\",\"/1f926fd7-6a2f-4b9b-b00f-ac06c4958b6b/2f989fd2-bb56-4d76-809e-b672000fa9f7\"],\"unreferencedTimestampMs\":1646688495162},\"/af627ff8-30d1-4e89-8220-12c59b99016e/root\":{\"outboundRoutes\":[\"/af627ff8-30d1-4e89-8220-12c59b99016e/d2dba981-eeb5-4fdc-aa91-8549d108f8bd\",\"/af627ff8-30d1-4e89-8220-12c59b99016e\"],\"unreferencedTimestampMs\":1646688495162},\"/af627ff8-30d1-4e89-8220-12c59b99016e/d2dba981-eeb5-4fdc-aa91-8549d108f8bd\":{\"outboundRoutes\":[\"/af627ff8-30d1-4e89-8220-12c59b99016e\"],\"unreferencedTimestampMs\":1646688495162},\"/af627ff8-30d1-4e89-8220-12c59b99016e\":{\"outboundRoutes\":[\"/af627ff8-30d1-4e89-8220-12c59b99016e/root\",\"/af627ff8-30d1-4e89-8220-12c59b99016e/d2dba981-eeb5-4fdc-aa91-8549d108f8bd\"],\"unreferencedTimestampMs\":1646688495162},\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86/root\":{\"outboundRoutes\":[\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86/a0f98d43-9e0c-41d1-9bee-e72fd5407d90\",\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86\"],\"unreferencedTimestampMs\":1646688495162},\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86/a0f98d43-9e0c-41d1-9bee-e72fd5407d90\":{\"outboundRoutes\":[\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86\"],\"unreferencedTimestampMs\":1646688495162},\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86\":{\"outboundRoutes\":[\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86/root\",\"/1f3a00f9-e6ce-40c1-a206-1650c1280a86/a0f98d43-9e0c-41d1-9bee-e72fd5407d90\"],\"unreferencedTimestampMs\":1646688495162},\"/b194361b-3b4b-4623-b7b9-fe5857095751/root\":{\"outboundRoutes\":[\"/b194361b-3b4b-4623-b7b9-fe5857095751/bc03124d-5ee3-4a74-b3bc-fd2c5f93b954\",\"/b194361b-3b4b-4623-b7b9-fe5857095751\"],\"unreferencedTimestampMs\":1646688495162},\"/b194361b-3b4b-4623-b7b9-fe5857095751/bc03124d-5ee3-4a74-b3bc-fd2c5f93b954\":{\"outboundRoutes\":[\"/b194361b-3b4b-4623-b7b9-fe5857095751\"],\"unreferencedTimestampMs\":1646688495162},\"/b194361b-3b4b-4623-b7b9-fe5857095751\":{\"outboundRoutes\":[\"/b194361b-3b4b-4623-b7b9-fe5857095751/root\",\"/b194361b-3b4b-4623-b7b9-fe5857095751/bc03124d-5ee3-4a74-b3bc-fd2c5f93b954\"],\"unreferencedTimestampMs\":1646688495162},\"/\":{\"outboundRoutes\":[\"/_scheduler\",\"/defaultComponent\"]}}}",
```

Comment:
I'm hoping this GC blob is in the same order as the previous one

Reply:
I just did a JSON diff, and it seems the unreferenced timestamps are not the same

Reply:
Is unreferencedTimestamp the only thing that changed?

Reply:
Yeah, it's the only thing that I saw. My JSON converter ordered the two things alphabetically. It went from one value to the next. It seems like a relatively small change

Reply:
It's about a 17 second difference between the two values

Reply:
Okay cool, thanks!